Controller/SuggestionProviderController.php

```diff
@@ -86,4 +103,38 @@ public function addSuggestionProvider(SuggestionProviderInterface $matcher, $gro
     {
         $this->suggestionProviders[] = array('provider' => $matcher, 'group' => $group);
     }
+
+    /**
+     * You can define your templates for each format in the bundle's configuration.
+     *
+     * @param string $format
+     *
+     * @return string
+     */
+    private function getTemplateForSuggestions($format = 'html')
+    {
+        if (!isset($this->configuration['templates'][$format])) {
+            throw new InvalidConfigurationException(
```

Comment:
should we not revert to the parent in this case? or return false and let the calling method go to parent::showAction even?